query/sorting.go

```diff
@@ -29,7 +30,10 @@ func (c SortCriteria) GoString() string {
 // See: https://github.com/infobloxopen/atlas-app-toolkit#sorting
 func ParseSorting(s string) (*Sorting, error) {
 	var sorting Sorting
-
+	sortingRegexp := regexp.MustCompile(`^[A-Za-z0-9_,.\- ]+$`)
+	if !sortingRegexp.MatchString(s) {
+		return nil, fmt.Errorf("invalid character in the sort criteria")
+	}
```

Comment:
I see no explanation here as to why this is being checked https://github.com/infobloxopen/atlas-app-toolkit/tree/master/query#sorting 
Why is this needed?

Reply:
The purpose of these changes is to exclude the use of special characters in the sorting criteria. Otherwise, a similar validation must be used in every List method where a * query.Sorting object is used. For example, such a request .../v2/users? _Limit5 & offset = 5 & order_by = <string> will return an error like "internal-error pq: syntax error at or near" <"". But I would like to have the error "Invalid request"